wordle-web/src/App.vue

```diff
@@ -1,86 +1,45 @@
-<template>
-  <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
-    <div class="wrapper">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">Class</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-    </div>
-  </header>
-
-  <RouterView />
-</template>
-
 <script setup lang="ts">
-import { RouterLink, RouterView } from 'vue-router'
-import HelloWorld from './components/HelloWorld.vue'
+import { RouterView } from 'vue-router'
+import ThemeSelector from './components/ThemeSelector.vue'
 </script>
 
-<style scoped>
-header {
-  line-height: 1.5;
-  max-height: 100vh;
-}
-
-.logo {
-  display: block;
-  margin: 0 auto 2rem;
-}
-
-nav {
-  width: 100%;
-  font-size: 12px;
-  text-align: center;
-  margin-top: 2rem;
-}
-
-nav a.router-link-exact-active {
-  color: var(--color-text);
-}
-
-nav a.router-link-exact-active:hover {
-  background-color: transparent;
-}
-
-nav a {
-  display: inline-block;
-  padding: 0 1rem;
-  border-left: 1px solid var(--color-border);
-}
-
-nav a:first-of-type {
-  border: 0;
-}
-
-@media (min-width: 1024px) {
-  header {
-    display: flex;
-    place-items: center;
-    padding-right: calc(var(--section-gap) / 2);
-  }
-
-  .logo {
-    margin: 0 2rem 0 0;
-  }
-
-  header .wrapper {
-    display: flex;
-    place-items: flex-start;
-    flex-wrap: wrap;
-  }
-
-  nav {
-    text-align: left;
-    margin-left: -1rem;
-    font-size: 1rem;
-
-    padding: 1rem 0;
-    margin-top: 1rem;
-  }
-}
-</style>
+<template>
+  <v-container fluid class="pa-0">
+    <v-row class="ma-0">
+      <v-col class="pa-0">
+        <v-toolbar density="compact" class="elevation-4">
+          <v-toolbar-title style="cursor: pointer" @click="$router.push('/')">
+            <v-icon icon="mdi-gamepad-square" class="mx-1" color="teal"></v-icon>
+            Wordle
+          </v-toolbar-title>
+          <v-spacer></v-spacer>
+          <v-menu location="bottom">
+            <template v-slot:activator="{ props }">
+              <v-btn k v-bind="props"> <v-icon icon="mdi-menu" size="x-large"></v-icon> </v-btn>
+            </template>
+
+            <v-list flat>
+              <v-list-item>
+                <v-btn variant="plain" to="/"> Home </v-btn>
+              </v-list-item>
+              <v-list-item>
+                <v-btn variant="plain" to="/about"> About </v-btn>
+              </v-list-item>
+              <v-list-item>
+                <v-btn variant="plain" to="/game"> Game </v-btn>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+
+          <ThemeSelector />
+        </v-toolbar>
+      </v-col>
+    </v-row>
+
+    <v-row>
```

Comment:
Same comment here, don't use v-row and v-col when you aren't configuring a grid system